### PR TITLE
Update login.js

### PR DESCRIPTION
**Pull request by Rhitu29**
Show today's time

### CODE REVIEW

File: cypress/integration/login.js [hunk 1]
```diff
@@ -20,5 +20,9 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
         cy.get('.main-nav').find('li').should('have.length', 1)
+
+    })
+    it('Should show current date', function (){
+        cy.get('.calendar--selected').contains('13')
```
Comment (MonkeviciuteGerda) [inline cypress/integration/login.js]: will it work tomorrow? Think about how you can make it work everyday.

**Review by MonkeviciuteGerda — CHANGES_REQUESTED** [review]

File: cypress/integration/login.js (hunk 1 above)
Comment (Rhitu29) [reply cypress/integration/login.js]: Request you to review again. :)

File: cypress/integration/login.js [hunk 2]
```diff
@@ -20,5 +20,10 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
         cy.get('.main-nav').find('li').should('have.length', 1)
+
+    })
+    it('Should show current date', function (){
+        var date = new Date();
+        cy.get('.calendar--selected').contains(date.getDate())
```
Comment (linukkis) [inline cypress/integration/login.js:27]: That's the stuff! :) 👍

**Review by linukkis — APPROVED** [review]

**Review by linukkis — COMMENTED** [review]
Turns out there are conflicts in your pull-request, meaning that the new changes of this pull request were made in the same place as some other changes before that. Please make sure to resolve the conflict and if you have trouble doing that, ask and we'll help you :)

**Pull request merged**